### DISCUSSION TITLE
Modify the background of the slider for custom skin to be flat

### DISCUSSION THREAD

**Discussion by RQNG** [ppy/osu#25812] [Ideas] [answered]
I really like the style of the default skin slider, which has a flat background like this:
<img width="355" alt="Snipaste_2023-12-18_14-10-23" src="https://github.com/ppy/osu/assets/115459678/cbf5fd77-eccd-4f3e-aacb-db6f7a7ff219">
But I can't achieve this effect with custom skin, it looks like a curved groove.
<img width="252" alt="Snipaste_2023-12-18_14-14-52" src="https://github.com/ppy/osu/assets/115459678/c01e98c6-b296-4ce5-a2fa-c9ca02b953da">
I hope to add this feature to custom skins.

**Comment by RQNG** [comment]
Does anyone care about this design? If the creator thinks it's modern, why not add this effect to the custom skin?

**Reply by peppy** [reply, marked as answer]
yes it may happen one day
